lib/workers/repository/process/extract-update.ts

```diff
@@ -9,16 +9,17 @@ import { PackageFile } from '../../../manager/common';
 import { RenovateConfig } from '../../../config';
 import { BranchConfig } from '../../common';
 
-export type ExtractAndUpdateResult = {
-  res: WriteUpdateResult | undefined;
+export type ExtractResult = {
   branches: BranchConfig[];
   branchList: string[];
   packageFiles?: Record<string, PackageFile[]>;
```

Comment:
```suggestion
  packageFiles: Record<string, PackageFile[]>;
```

Reply:
Sorry, I'm against it. https://github.com/renovatebot/renovate/pull/5835#discussion_r407385002

Reply:
I don't agree, but if they're ignored them remove them completely?

Reply:
This is ignored in one place and in use in another. And that how it works in master right now, without that PR.

If this is a bug, lets fix it in different PR.